### ISSUE TITLE
Will you provide the code for calculating the evaluation metrics?

### ISSUE THREAD

**Issue by jiwon1202** [whwjdqls/DEEPTalk#9]
I am curious about how you calculated FID and Emo-FID in detail.

**Comment by whwjdqls** [comment]
Hi jiwon, thanks for the attention.

For FID, we used the well known open-sourced codebase for evaluating image generation using pretrained image classification models (trained on ImageNet).

We rendered all the faces for the test set, and compared its feature distribution with the GT test set, as one would do for image generation. 

For Emo-FID, we simply replaced the image classification model with an emotion classification model (trained on AffectNet) to extract features.

Feel free to ask any questions.
